$NetBSD: patch-aa,v 1.5 2016/12/10 21:46:52 dholland Exp $

Fix configure with png>1.2.
Remove references to ancient builtin copy of curl.

--- configure.orig	2006-07-16 01:48:28.000000000 +0000
+++ configure
@@ -272,7 +272,6 @@ PACKAGE_STRING=
 PACKAGE_BUGREPORT=
 
 ac_unique_file="game/main.cpp"
-ac_subdirs_all="$ac_subdirs_all curl"
 ac_subst_vars='SHELL PATH_SEPARATOR PACKAGE_NAME PACKAGE_TARNAME PACKAGE_VERSION PACKAGE_STRING PACKAGE_BUGREPORT exec_prefix prefix program_transform_name bindir sbindir libexecdir datadir sysconfdir sharedstatedir localstatedir libdir includedir oldincludedir infodir mandir build_alias host_alias target_alias DEFS ECHO_C ECHO_N ECHO_T LIBS build build_cpu build_vendor build_os host host_cpu host_vendor host_os target target_cpu target_vendor target_os INSTALL_PROGRAM INSTALL_SCRIPT INSTALL_DATA CYGPATH_W PACKAGE VERSION ACLOCAL AUTOCONF AUTOMAKE AUTOHEADER MAKEINFO install_sh STRIP ac_ct_STRIP INSTALL_STRIP_PROGRAM mkdir_p AWK SET_MAKE am__leading_dot AMTAR am__tar am__untar CC CFLAGS LDFLAGS CPPFLAGS ac_ct_CC EXEEXT OBJEXT DEPDIR am__include am__quote AMDEP_TRUE AMDEP_FALSE AMDEPBACKSLASH CCDEPMODE am__fastdepCC_TRUE am__fastdepCC_FALSE CXX CXXFLAGS ac_ct_CXX CXXDEPMODE am__fastdepCXX_TRUE am__fastdepCXX_FALSE RANLIB ac_ct_RANLIB APPLE_TRUE APPLE_FALSE WIN32_TRUE WIN32_FALSE USE_LZMA_TRUE USE_LZMA_FALSE subdirs LIBOBJS LTLIBOBJS'
 ac_subst_files=''
 
@@ -3864,13 +3863,13 @@ else
     LIBS="-L/usr/local/lib -L/usr/X11R6/lib $LIBS"
     CXXFLAGS="$CXXFLAGS -I/usr/X11R6/include"
 
-echo "$as_me:$LINENO: checking for main in -lpng12" >&5
-echo $ECHO_N "checking for main in -lpng12... $ECHO_C" >&6
-if test "${ac_cv_lib_png12_main+set}" = set; then
+echo "$as_me:$LINENO: checking for main in -lpng" >&5
+echo $ECHO_N "checking for main in -lpng... $ECHO_C" >&6
+if test "${ac_cv_lib_png_main+set}" = set; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lpng12  $LIBS"
+LIBS="-lpng  $LIBS"
 cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
@@ -3908,25 +3907,25 @@ if { (eval echo "$as_me:$LINENO: \"$ac_l
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; }; then
-  ac_cv_lib_png12_main=yes
+  ac_cv_lib_png_main=yes
 else
   echo "$as_me: failed program was:" >&5
 sed 's/^/| /' conftest.$ac_ext >&5
 
-ac_cv_lib_png12_main=no
+ac_cv_lib_png_main=no
 fi
 rm -f conftest.err conftest.$ac_objext \
       conftest$ac_exeext conftest.$ac_ext
 LIBS=$ac_check_lib_save_LIBS
 fi
-echo "$as_me:$LINENO: result: $ac_cv_lib_png12_main" >&5
-echo "${ECHO_T}$ac_cv_lib_png12_main" >&6
-if test $ac_cv_lib_png12_main = yes; then
+echo "$as_me:$LINENO: result: $ac_cv_lib_png_main" >&5
+echo "${ECHO_T}$ac_cv_lib_png_main" >&6
+if test $ac_cv_lib_png_main = yes; then
   cat >>confdefs.h <<_ACEOF
 #define HAVE_LIBPNG12 1
 _ACEOF
 
-  LIBS="-lpng12 $LIBS"
+  LIBS="-lpng $LIBS"
 
 else
   { { echo "$as_me:$LINENO: error: libpng is needed" >&5
@@ -4174,7 +4173,6 @@ fi
 
 
 
-subdirs="$subdirs curl"
 
 
                                                                                           ac_config_files="$ac_config_files Makefile utils/Makefile utilssdl/Makefile utilsgl/Makefile game/Makefile tools/Makefile data/Makefile data/music/Makefile tinyxml/Makefile"
